src/Athena/Views/Home/Schedule_Page.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿
+@{
+    ViewData["Title"] = "Schedule_Page";
+}
+
+<h2>Schedule</h2>
+
+
+
+
+<div id="name"><ul></ul></div>
+
+
+
+
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
```

Comment:
Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.